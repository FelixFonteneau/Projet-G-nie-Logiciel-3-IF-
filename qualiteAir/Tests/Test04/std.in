3
2
01/01/2017 00:00
01/01/2017 01:00
7
