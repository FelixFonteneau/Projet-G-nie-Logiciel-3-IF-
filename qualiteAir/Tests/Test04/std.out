Bonjour, et bienvenue dans cette application de surveillance de la qualité de l'air.
Le service est en cours d'initialisation, veuillez patientez...

Nombre de mesures analysées : 80

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


L'indice ATMO pour l'endroit selectionné vaut 10 et a été calculé à l'aide de 3 capteurs dans un rayon de 38 km.

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


L'indice ATMO pour l'endroit selectionné vaut 3 et a été calculé à l'aide de 3 capteurs dans un rayon de 60 km.

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


L'indice ATMO pour l'endroit selectionné vaut 3 et a été calculé à l'aide de 2 capteurs dans un rayon de 39 km.

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


L'indice ATMO pour l'endroit selectionné vaut 10 et a été calculé à l'aide de 1 capteur dans un rayon de 42 km.

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


Il n'y a pas de capteurs dans un rayon de 90 km autour de l'endroit selectionné.

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Merci et à bientôt !

